packages/wesley-cli/src/commands/models.mjs

```
@@ -29,15 +25,12 @@ export class ModelsCommand extends GeneratorCommand {
     const result = await generator.generate(ir, { outDir: options.outDir });
 
     if (!options.quiet) {
-      console.log('✨ Generated model classes:');
-      result.files.forEach((file) => console.log(`  ✓ ${file}`));
-      console.log(`\n📁 Target: ${result.target} (${result.outputDir})`);
+      console.log('Generated model classes:');
+      result.files.forEach((file) => console.log(`  ${file}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)